android/src/main/kotlin/com/abilia/system_settings_editor/SystemSettingsHandler.kt

```diff
@@ -103,9 +103,9 @@ class SystemSettingsHandler(private val context: Context) {
   }
 
   internal fun setScreenOffTimeoutHandler(call: MethodCall, result: MethodChannel.Result) {
-    val timeout: Int? = call.argument("timeout")
+    val timeout: Long? = call.argument("timeout")
     timeout?.let {
-      setScreenOffTimeout(it)
+      setScreenOffTimeout(it.toInt())
```

Comment:
Couldn't `setScreenOffTimeout`just take a Long now?
Also, increase version number!

Reply:
Valid. Also forgot to push the tag.